README.md

```diff
@@ -7,15 +7,24 @@ Evaluating machine learning models is an important aspect of building models. Th
 
 For instance, in order to map a logistic regression value to a binary category, you must define a classification threshold (also called the decision threshold). In say a cancer patient classification, a value above that threshold indicates "Patient has cancer"; a value below indicates "Patient does not have cancer." It is tempting to assume that the classification threshold should always be 0.5, but thresholds are problem-dependent, and are therefore values that you must tune.
```

Comment:
'logistic regression value' should be 'probability representation from logistic regression'

Reply:
Got it.